daemon/daemon_linux.go

```diff
@@ -2,16 +2,30 @@ package daemon // import "github.com/docker/docker/daemon"
 
 import (
 	"bufio"
```

Comment:
leaving _unix.go as-is and adding `//build +linux +freebsd` would be nicer.

Reply:
I don't think we can do that. We are using common functions within  daemon_unix.go for both daemon_darwin.go and daemon_linux.go. We decoupled them like _darwin.go and _linux.go since we would have errors such as similar function definition in both the files.